.dockerignore

```diff
@@ -1,5 +1,4 @@
 # Azure Functions artifacts
-bin
```

Comment:
What's this change for?

Reply:
Please ignore this change, reverted it. `bin` folder at /OOProcSmokeTests/durableJava/build/azure-functions/durableJava cannot be copied from the pipeline to the docker image. I was trying to test if changing the dockerignore would allow it.